test/integration/vaults/roles.integration.ts

```diff
@@ -326,25 +378,97 @@ describe("Integration: Staking Vaults Dashboard Roles Initial Setup", () => {
 
         it("mintShares", async () => {
           await testMethod(
+            dashboard,
             "mintShares",
             {
               successUsers: [roles.minter, owner],
-              failingUsers: Object.values(roles).filter((r) => r !== roles.minter && r !== owner),
+              failingUsers: [...Object.values(roles).filter((r) => r !== roles.minter), nodeOperatorManager, stranger],
             },
             [stranger, 100n],
             await dashboard.MINT_ROLE(),
           );
         });
 
+        it("burnShares", async () => {
+          // Mint shares to all users - balance check happens before ACL check
+          const sharesAmount = 100n;
+          const allUsers = [...Object.values(roles), owner, nodeOperatorManager, stranger];
+          await dashboard.connect(roles.minter).mintShares(roles.minter, sharesAmount * BigInt(2 * allUsers.length));
+          const stethAmount = await ctx.contracts.lido.getPooledEthByShares(sharesAmount);
+          for (const user of allUsers) {
+            await ctx.contracts.lido.connect(roles.minter).transferShares(user, stethAmount);
```

Comment:
Should not be shares?
```suggestion
            await ctx.contracts.lido.connect(roles.minter).transferShares(user, sharesAmount);
```